js/controller.js

```diff
@@ -673,10 +675,12 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
      */
     startBufferingTimer: function() {
       this.stopBufferingTimer();
+      var bufferingSpinnerDelay = Utils.getPropertyValue(this.skin.props.skinConfig, 'general.bufferingSpinnerDelay');
```

Comment:
Should we still have a default in case it gets deleted/omitted from skin.json?

Reply:
This should have a value because the custom skin is merged with the default one. `constrainToRange` will give it a default value of 0 if this was undefined for some reason though.